src/themes/default/components/core/blocks/Checkout/Payment.vue

```diff
@@ -6,7 +6,7 @@
           class="number-circle lh35 cl-white brdr-circle align-center weight-700"
           :class="{ 'bg-cl-th-accent' : isActive || isFilled, 'bg-cl-tertiary' : !isFilled && !isActive }"
         >
-          3
```

Comment:
Would be better to have this conditional statement directly in the {{}} using for example ternary operator (condition ? Label1 : Label2)

Reply:
Please review the latest commit - 3021813